internal/export/exportfile.go

```diff
@@ -176,6 +176,8 @@ func assignReportType(reportType *string, pbek *export.TemporaryExposureKey) {
 	}
 }
 
+// The batch num and batch size are always set to 1/1.
```

Comment:
Maybe clarify you mean in the output, as opposed to the function assumes the inputs are 1/1?